src/connections.rs

```diff
@@ -43,76 +39,71 @@ impl Connection {
         self.quic_conn.remote_address()
     }
 
-    /// Send message to peer and await for a reponse.
-    pub async fn send(&mut self, msg: Bytes) -> Result<Bytes> {
+    /// Get connection streams for reading/writing
+    pub async fn create_connection_streams(&self) -> Result<ConnectionStreams> {
```

Comment:
I was thinking we may want to return two separate objects here so the user can spawn tasks to send and receive on the bi-stream from separate tasks/threads.

Reply:
Yes that makes sense :+1: